source/AMDSBufferGroup.cpp

```diff
@@ -67,11 +73,28 @@ void AMDSBufferGroup::populateData(AMDSClientStartTimePlusCountDataRequest* clie
 			AMDSFlatArray oneFlatArray;
 			dataHolder->data(&oneFlatArray);
 			clientStartTimePlusCountDataRequest->appendData(dataHolder);
-//			request->histogramData()->append(dataHolders_[iCurrent]);
 		}
 	}
 }
 
+void AMDSBufferGroup::populateData(AMDSClientRelativeCountPlusCountDataRequest* clientDataRequest)
+{
+	int relativeCount = clientDataRequest->relativeCount();
+	int count = clientDataRequest->count();
+
+	int startIndex = dataHolders_.count() - 1 - relativeCount;
+	int endIndex = dataHolders_.count() - 1 - relativeCount + count;
```

Comment:
Conceptually, would this make more sense to just use `int endIndex = startIndex + count;`?  Makes it a bit clearer.